Automattic/wp-calypso
# A4A: use proper getActiveAgencyId selector

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/207

## Proposed Changes

It appears that we already have `getActiveAgencyId` selector so I just updated code to use it.

## Testing Instructions



- Check the current branch from live link `sites` pages and confirm they work normally.

## Pre-merge Checklist
